tests/test_serializer_fields.py

```diff
@@ -0,0 +1,62 @@
+from dateutil.relativedelta import relativedelta
+from rest_framework import serializers, viewsets
+
+from vng_api_common.serializers import DurationField
+
+
+class DurationSerializer(serializers.Serializer):
+    duration = DurationField()
+
+
+class DurationView(viewsets.ModelViewSet):
+    serializer_class = DurationSerializer
+
+
+def test_duration_field_internal_data():
+    data = {"duration": "P10D"}
+    serializer = DurationSerializer(data={"duration": "P10D"})
+    assert serializer.is_valid()
+    assert serializer.to_internal_value(data) == {"duration": relativedelta(days=+10)}
+
+
+def test_duration_to_representation():
+    data = {"duration": "P10D"}
+    serializer = DurationSerializer(data=data)
+    assert serializer.is_valid()
+    assert serializer.to_representation({"duration": relativedelta(days=+10)}) == data
+
+
+def test_valid_duration_field():
+    duration_options = [
+        "P10D",
+        "P1M10D",
+        "P1Y1M10D",
+        "P-10D",
+        "P-1M-10D",
+        "P-1Y-1M-10D",
+        "P1Y-1M10D",
+    ]
+    data = {"duration": ""}
+    for d in duration_options:
+        data["duration"] = d
+        serializer = DurationSerializer(data=data)
+        assert serializer.is_valid()
```

Comment:
Adding an assert for serializer.to_internal_value would make sure that everything is correct.